src/utils/sdfg/json_serializer.ts

```diff
@@ -17,11 +19,13 @@ function reviver(name: string, val: unknown) {
 }
 
 function isDict(v: unknown): v is Record<string, unknown> {
-    return typeof v === 'object' && v !== null && !(v instanceof Array) && !(v instanceof Date);
+    return typeof v === 'object' && v !== null && !(v instanceof Array) &&
+        !(v instanceof Date);
 }
 
-function replacer(name: string, val: unknown, orig_sdfg: unknown): unknown {
-    if (val && isDict(val) && val !== orig_sdfg && 'type' in val && val.type === 'SDFG') {
+function replacer(_name: string, val: unknown, orig_sdfg: unknown): unknown {
+    if (val && isDict(val) && val !== orig_sdfg && 'type' in val &&
+        val.type === 'SDFG') {
```

Comment:
why are there so many `unknown`s?

Reply:
These functions are either for determining an unknown type (isDict) or incredibly generic and used by `JSON.stringify` (replacer). I do not know what the types for the latter would be.